eventstore/tasks.py

```diff
@@ -532,3 +538,75 @@ def process_ada_assessment_notification(
     triage.risk = triage.calculate_risk()
     triage.full_clean()
     triage.save()
+
+
+@app.task(
+    autoretry_for=(RequestException, SoftTimeLimitExceeded, TembaHttpError),
+    retry_backoff=True,
+    max_retries=15,
+    acks_late=True,
+    soft_time_limit=10,
+    time_limit=15,
+)
+def post_random_contacts_to_slack_channel():
+    # Get 10 random contacts to post to slack channel
+    if settings.RAPIDPRO_URL and settings.RAPIDPRO_TOKEN:
+        rapidpro_url = urljoin(settings.RAPIDPRO_URL, "/contact/read/{}/")
+
+        contact_details = []
+
+        while len(contact_details) < settings.RANDOM_CONTACT_LIMIT:
+            profile_link, contact_uuid = get_random_contact()
+
+            rapidpro_link = rapidpro_url.format(contact_uuid)
+            links = {"Rapid Pro Link: ": rapidpro_link, "Turn Link": profile_link}
+
+            contact_details.append(links)
+
+        if contact_details:
+            sent = send_slack_message("test-mon", str(contact_details))
```

Comment:
I'm not sure this `str()` of a dict will look great in slack, we'll need to do some formatting.

and lets make the channel configurable with an environment variable. we can also use this to disable the task for QA etc